app/views/comments/_comments.html.slim

```diff
@@ -1,5 +1,6 @@
--if user_signed_in?
-  = render 'comments/form', commentable: commentable
+-if current_user
```

Comment:
after last changes on master this is not right.  Use "if user_signed_in?"  instead of "if current-user"